content/docs/reference/compatibility.md

```diff
@@ -24,7 +24,7 @@ Roles created in the Neon console, CLI, or API, including the default role creat
 
 Roles created in Neon with SQL syntax, from a command-line tool like `psql` or the [Neon SQL Editor](/docs/connect/query-with-psql-editor), have the same privileges as newly created roles in a standalone Postgres installation. These roles are not granted membership in the `neon_superuser` role. You must grant these roles the privileges you want them to have. For more information, see [Manage roles with SQL](/docs/manage/roles#manage-roles-with-sql).
 
-Neon roles cannot access replication methods or install Postgres extensions other than those supported by Neon.
+Neon roles install Postgres extensions other than those supported by Neon.
```

Comment:
Is there a word missing? "roles cannot install"?

Reply:
Fixed. Thank you!